src/Util/constants.js

```diff
@@ -0,0 +1,90 @@
+const MENU_KIND = Object.freeze({
+  APPETIZER: '애피타이저',
+  MAIN: '메인',
+  DESSERT: '디저트',
+  DRINK: '음료',
+});
+
+const MENU = Object.freeze({
+  양송이수프: { price: 6000, kind: MENU_KIND.APPETIZER },
+  타파스: { price: 5500, kind: MENU_KIND.APPETIZER },
+  시저샐러드: { price: 8000, kind: MENU_KIND.APPETIZER },
+  티본스테이크: { price: 55000, kind: MENU_KIND.MAIN },
+  바비큐립: { price: 54000, kind: MENU_KIND.MAIN },
+  해산물파스타: { price: 35000, kind: MENU_KIND.MAIN },
+  크리스마스파스타: { price: 25000, kind: MENU_KIND.MAIN },
+  초코케이크: { price: 15000, kind: MENU_KIND.DESSERT },
+  아이스크림: { price: 5000, kind: MENU_KIND.DESSERT },
+  제로콜라: { price: 3000, kind: MENU_KIND.DRINK },
+  레드와인: { price: 60000, kind: MENU_KIND.DRINK },
+  샴페인: { price: 25000, kind: MENU_KIND.DRINK },
+});
+
+const DATE = Object.freeze({
+  EVENT_START: 1,
+  EVENT_END: 25,
+  PERIOD_DISCOUNT: 1000,
+  SPECIAL_DISCOUNT: 1000,
+  SPECIAL_DATE: [3, 10, 17, 24, 25, 31],
+  PER_DAY_DISCOUNT: 100,
+});
+
+const INFO = Object.freeze({
+  UNIT: '원',
+  COUNT: '개',
+  WEEK_DISCOUNT: 2023,
+  ORDER_MINIMUM: 10000,
+  GIFT_CONDITION: 120000,
```

Comment:
이번에 금액에 관한 숫자를 다루면서 `numeric seperator` 라는 방식을 알게되었는데
개인적으로 숫자 단위가 한눈에 들어와서 좋더라구요!

```suggestion
  GIFT_CONDITION: 120_000,
```

Reply:
오 금액이 큰 경우 더 유용하겠네요 정보 감사합니다 !